includes/base_controls/QControlProxy.class.php

```diff
@@ -90,7 +142,7 @@ public function __set($strName, $mixValue) {
 				
 				case 'TargetControlId': 
 					try {
-						return ($this->strFoo = QType::Cast($mixValue, QType::String));
```

Comment:
No idea mate! :stuck_out_tongue: I wanted to ask the same when I saw this. But there were so many things so I just corrected that and moved ahead.